### DISCUSSION TITLE
adding bunch of styles at a specific breakpoint without prefixing all of them with the breakpoint name

### DISCUSSION THREAD

**Discussion by DjamelAbbou** [tailwindlabs/tailwindcss#10578] [Help]
hey everyone, hope u doing fine, i wanted to ask if there is a way to add a bunch of styles in tailwindcss at a specific breakpoint at once without having to prefix all of the style names with the breakpoint name.

**Comment by wongjn** [comment]
There is no such feature in Tailwind.

There have been many suggestions about adding such a feature, but [here's one such response from a maintainer](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

[Also another thread here](https://github.com/tailwindlabs/tailwindcss/discussions/8337) with more responses from maintainers.
